tests/Custom/DesiredCapabilitiesTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Behat\Mink\Tests\Driver\Custom;
+
+use Behat\Mink\Driver\Selenium2Driver;
+use Behat\Mink\Tests\Driver\TestCase;
+
+class desiredCapabilitieTest extends TestCase
+{
+
+    public function testGetDesiredCapabilties()
+    {
+        $driver = $this->getSession()->getDriver();
+        $this->assertNotEmpty($driver->getDesiredCapabilities(), 'desiredCapabilities empty');
```

Comment:
Sounds good.  I do have a question about this: for those first 2 tests you mention, do you mean to call `setDesiredCapabilities()` with some set of capabilities, then call `getDesiredCapabilities()` to verify what we get back is what we set?  If so, can you tell me how I can call `setDesiredCapabilities()` within in the test case?  From what I can tell, I would have to call `$this->getSession()->getDriver()->setDesiredCapabilities()`, but in doing so starts the session and thus makes setting the desiredCapabilities impossible/moot.  Is there somewhere else I need to call setDesiredCapabilities() during setup?

Again, I'm no expert on this, so any guidance you can provide would be helpful!